docs/jargon/mesh.md

```diff
@@ -80,6 +92,86 @@ An [**Activity**](#activity) that is invoked directly in the same process by Wor
 
 - Although a Local Activity consumes less resources than a normal [**Activity**](#activity), it is subject to shorter durations and a lack of rate limiting.
 
+## Retry Policy
+
+A collection of attributes that instructs the Temporal Server how to retry a failure of an [Activity Task Execution](#activity-task-execution) or a [Workflow Execution](#workflow-execution).
+
+- If a custom Retry Policy is to be used, it must be provided as an options parameter when an [Activity Execution](#activity-execution) or [Workflow Execution](#workflow-execution) is invoked.
+
+- The time between a retry is the _retry interval_.
```

Comment:
I previously had "The time between retries...", but that leaves out the first retry interval, which is between the initial attempt and the first retry attempt. My next commit changes the phrase to "The wait time before a retry...."